SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+
+## Reporting a Vulnerability
+
+You can report a vulnerability using GitHub's private reporting
+feature:
+
+1. Go to [github.com/dyninst/dyninst/security](https://github.com/dyninst/dyninst/security).
```

Comment:
It looks like that needs some configuration before it will work. In particular, the "Report a Vulnerability" isn't enabled for the repo. I don't have any objections to turning it on.